### PR TITLE
initial readme for setting up

### PR DESCRIPTION
**Pull request by jinpark**
@RachellCalhoun if this looks good, comment below and press the green button!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+-----
+
+This assumes that you have python3 installed. (we are using 3.4 but this should work for most python 3 versions)
+
+1. Clone the project by entering `git clone git@github.com:MOOCCircle/codeforeveryone.git` in the terminal
+    - This will create a folder called `codeforeveryone` where you ran the command
+
+2. Go into the `codeforeveryone` directory `cd codeforeveryone`
+
+3. Create a virtualenv `venv codeforeveryone`
```
Comment (hassanabidpk) [inline readme.md]: How about using pyvenv for creating virtual environment ?  Reference : https://docs.python.org/3/library/venv.html

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Setting up your local environment
+-----
+
+This assumes that you have python3 installed. (we are using 3.4 but this should work for most python 3 versions)
+
+1. Clone the project by entering `git clone git@github.com:MOOCCircle/codeforeveryone.git` in the terminal
+    - This will create a folder called `codeforeveryone` where you ran the command
+
+2. Go into the `codeforeveryone` directory `cd codeforeveryone`
```
Comment (hassanabidpk) [inline readme.md]: maybe create virtual env parallel to codeforeveryone folder ?

File: readme.md (hunk 1 above)
Comment (jinpark) [inline readme.md]: Thats why I mentioned in line 12, that you might have problems. `venv` worked for me but might not work on windows.

Comment (jinpark) [inline readme.md]: i added a note saying that you might have to use `pyvenv`

File: readme.md (hunk 2 above)
Comment (jinpark) [inline readme.md]: Yea that makes sense. I'll change it

File: readme.md (hunk 1 above)
Comment (jinpark) [inline readme.md]: Done. Can you take a look again?

**Comment by jinpark** [comment]
I would like @RachellCalhoun to read and accept/merge this.

**Comment by hassanabidpk** [comment]
looks fine to me now

**Comment by hassanabidpk** [comment]
@veroniquecho  try it in your free time too (But first install Python 3 from https://www.python.org/downloads/windows/

**Comment by jinpark** [comment]
Updated the readme with suggestions by @RachellCalhoun 

Mostly copy pasted stuff from the django girls handbook

**Pull request merged**